### PR TITLE
Add ligotools

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This adds a recipe for `ligotools`, an old (but still used) collection of utilities for LIGO

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@conda-forge/help-perl, one of the `outputs` in this recipe uses `perl`, but I think that the code is perl version independent. Is there `noarch: perl` equivalent, or should I just drop `perl` from the `requirements/host` list?

**Comment by duncanmmacleod** [comment]
@conda-forge/help-perl, I've written this recipe up as `noarch: generic` because the build just copies files from the tarball into the prefix. The windows tests are failing with the usual:

```
(%PREFIX%) %SRC_DIR%>tconvert now 
'tconvert' is not recognized as an internal or external command,
operable program or batch file.
```

What should I do to easily provide a windows-compatible executable for the `tconvert` perl script?

File: recipes/ligotools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    - perl-shebang.patch
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - tk
```
Comment (scopatz) [inline recipes/ligotools/meta.yaml]: Wait, tk is a build dep?

**Comment by scopatz** [comment]
If this is really noarch, then this should work on windows.  If you provide an entry point, an executable will be created for you

File: recipes/ligotools/meta.yaml (hunk 1 above)
Comment (duncanmmacleod) [reply recipes/ligotools/meta.yaml]: No, I'll fix that.

**Comment by duncanmmacleod** [comment]
> If this is really noarch, then this should work on windows. If you provide an entry point, an executable will be created for you

Does `perl` support entry points? If so, how do I use them...

**Comment by scopatz** [comment]
> Does perl support entry points? If so, how do I use them...

IIRC it does.  @jerowe probably knows off the top of her head :wink:

**Comment by duncanmmacleod** [comment]
I'm pretty sure most of this code won't run on windows anyway, so I will just skip that platform.

**Comment by scopatz** [comment]
It is probably worth asking on gitter

**Comment by duncanmmacleod** [comment]
ok, @conda-forge/staged-recipes, I've skipped windows builds for this package, and everything is ready to go, thanks in advance.

**Comment by chrisburr** [comment]
I think might better to make this `noarch` instead of skipping Windows.

**Comment by duncanmmacleod** [comment]
> I think might better to make this `noarch` instead of skipping Windows.

@chrisburr, I had `noarch: generic` before, but removed it because I couldn't get the windows builds to stop complaining. I would be happy to reintroduce it as long as the person with merge rights understands the windows failures are ok.

**Comment by chrisburr** [comment]
So long as nobody objects before the CI passes again I don't mind merging packages which have failures due to missing functionality on Windows because the upstream source doesn't support it.

**Comment by duncanmmacleod** [comment]
Ok, I've reverted my last commit, thanks @chrisburr.

**Pull request merged**